build-system/server/app.js

```diff
@@ -25,11 +25,11 @@ const bodyParser = require('body-parser');
 const cors = require('./amp-cors');
 const devDashboard = require('./app-index/index');
 const express = require('express');
+const fetch = require('node-fetch');
```

Comment:
Missing a few instances of request in this file

Reply:
Good catch. Updated them.